### DISCUSSION TITLE
Win10-64bit to Ubuntu 22 OBS has choppy audio

### DISCUSSION THREAD

**Discussion by repzikaros** [DistroAV/DistroAV#795] [Q&A - Support]
This was not an issue for me in OBS 27, but upgrading to the latest OBS and NDI version has caused choppy, almost peaking sounding audio. 
I've narrowed it down to NDI. Audio works fine on both pc's, but seems to get distorted arriving to my streaming pc (Ubuntu). 

I reinstalled Ubuntu and thought maybe it was some Linux thing I screwed up, but the problem still persists. 
Main PC is running Windows 10, 64-bit - OBS 28.1.2 (64-bit, windows) - obs-ndi-4.10.0-Qt6-Windows
Receiving PC is running Ubuntu 22.04.1 LTS - OBS 28.1.2 (linux) - obs-ndi-4.10.0-Ubuntu64
Here's a video of me going clinically insane trying to solve this for reference to what it sounds like.
https://www.twitch.tv/videos/1652880698

I noticed that there was another thread that seemed similar, but I made a new one since it's sending NDI to a different OS.

**Comment by Trouffman** [comment]
Closing old thread.
Reason : Using an old plugin/OBS version or old-issue.
